modules/op-release-notes-1-8.adoc

```diff
@@ -12,14 +12,276 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+
+* With this update, while deleting a pipeline run or task run, you can use both the `--keep` and `--keep-since` flags together.
+// link:https://github.com/tektoncd/cli/pull/1533[(#1533)]
+// Divyanshu Agrawal @divyansh42
+
+* With this update, you can use `Cancelled` as the value of the `spec.status` field rather than the deprecated `PipelineRunCancelled` value.
+// link:https://github.com/tektoncd/cli/pull/1554[(#1554)]
+// Vinamra Jain @vinamra28
+
+[id="operator-new-features-1-8_{context}"]
+=== Operator
+
+// [Hub]
+* With this update, as an administrator, you can configure your local {tekton-hub} instance to use a custom database rather than the default database.
+// link:https://issues.redhat.com/browse/SRVKP-2309[SRVKP-2309 Pipeline as code: validate repository URL uniqueness])
+// Puneet Punamiya
+
+* With this update, as a cluster administrator, if you enable your local {tekton-hub} instance, it periodically refreshes the database so that changes in the catalog appear in the {tekton-hub} web console. You can adjust the period between refreshes.
++
+Previously, to add the tasks and pipelines in the catalog to the database, you performed that task manually or set up a cron job to do it for you.
+// link:https://issues.redhat.com/browse/SRVKP-2252[SRVKP-2252 Refresh catalogs on schedule])
+// Puneet Punamiya
+
+* With this update, you can install and run a {tekton-hub} instance with minimal configuration. This way, you can start working with your teams to decide which additional customizations they might want.
+// link:https://issues.redhat.com/browse/SRVKP-2250[SRVKP-2250 Install Hub with default configs])
+// Shiv Verma
+
+[id="chains-new-features-1-8_{context}"]
+=== Tekton Chains
+
+:FeatureName: Tekton Chains
+include::snippets/technology-preview.adoc[]
+
+* With this update, you can log in to a vault by using OIDC rather than a static token. This change means that Spire can generate the OIDC credential so that only trusted workloads are allowed to log in to the vault. Additionally, you can pass the vault address as a configuration value rather than inject it as an environment variable.
+// link:https://github.com/tektoncd/chains/pull/369
+// Brandon Mitchell @sudo-bmitch or Shubham Minglani
+
+
+* The `chains-config` config map for {tekton-chains} in the `openshift-pipelines` namespace is automatically reset to default after upgrading the {pipelines-title} Operator because directly updating the config map is not supported when installed by using the {pipelines-title} Operator. However, with this update, you can configure {tekton-chains} by using the `TektonChain` custom resource. This feature enables your configuration to persist after upgrading, unlike the `chains-config` config map, which gets overwritten during upgrades.
+// link:https://issues.redhat.com/browse/SRVKP-2349
+// Piyush Garg @piyush-garg
+
+* This update adds `GIT_SSL_CAINFO` to the `git-clone` task so you can clone secured repositories.
```

Comment:
why is this in Chains category?

Reply:
Moved it to the Operator category. Thanks.